client-cli/src/command.rs

```diff
@@ -156,6 +156,13 @@ pub enum Command {
             case_insensitive = false
         )]
         hardware: Option<HardwareKind>,
+
```

Comment:
I think those arguments should be *optional* -- and the default value should be false (one will want to run the light client verification by default) ?

Reply:
ok, i'll change to option

Reply:
there is use case 
without genesis checking
and fast forward
-------------
default is
fast forward + genesis checking

Reply:
@leejw51crypto what about just disable genesis checking on fast forward? then no need for extra parameter -- just use "fast forward"?

Reply:
ok, i'll remove option,
skip genesis checking(lite client check) on fast forward mode